src/components/calculator/index.js

```diff
@@ -1,9 +1,8 @@
 import { getSetData } from "../../api";
 
-export const packEV = async (set) => {
+export const setEV = async (set) => {
     const url = `https://api.scryfall.com/cards/search?q=s%3A${set}+is%3Abooster`;
     const setData = await getSetData(url);
```

Comment:
I think this func has been renamed

Reply:
Yeah I renamed the function and updated the name in the other files where the function is being called